test/repository_test.rb

```diff
@@ -117,6 +118,34 @@
         end
       end
 
+      describe '.created_at' do
```

Comment:
By convention, the dot stands for class methods. In our case, we don't test any class method. Thus, please change the description to 'update created_at timestamp' and move this block under `.create` test case.